sign-up-task/src/components/SignUp/SignUpContainer.jsx

```diff
@@ -1,59 +1,56 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import {SignUpForm} from "./SignUpForm";
-import {fieldValidator} from "../../Utils/validators";
-
-export const SignUpContainer = () => {
-    const [values, setValues] = React.useState({
-        firstName:'',
-        lastName:'',
-        password:'',
-        confirmPassword:''
-    });
-    const [errors, setErrors] = React.useState(false);
-    const [helperText, setHelperText] = React.useState(false);
-
-
-    const handleChange = (prop) => (event) => {
-        event.persist();
-        setValues(values => ({...values, [prop]: event.target.value}));
-        let validation = fieldValidator(prop, event.target.value, values.password)
-        setErrors(values => ({...values, [prop]: validation.error}));
-        setHelperText(values => ({...values, [prop]: validation.helperText}));
+import { formValidator} from "../../Utils/validators";
+import {connect} from "react-redux";
+import {checkEmailDB, fieldOnChange, setClearForm} from "../../redux/signUp-reducer";
+import {
+    getEmailExist,
+    getErrors,
+    getHelperText,
+    getValues
+} from "../../redux/signUp-selectors";
+
+
+const SignUpContainer = (props) => {
```

Comment:
Container it's only your component wrapped by connect function, nothing more. Check my project for example